chaste_codegen/chaste_model.py

```diff
@@ -326,14 +334,28 @@ def _get_cytosolic_calcium_concentration_var(self):
             self._logger.info(warning)
             return cytosolic_calcium_concentration
 
+    def _get_modifiers(self):
+        """ Get the variables that can be used as modifiers, if us_modifiers is switched on.
+
+        These are all variables with annotation (including stat vars)
+        except the stimulus current and time (the free variable)"""
+        modifiers = []
+        if self.use_modifiers:
+            modifiers = [m for m in self._model.variables()
```

Comment:
Minor optimisation would be to start with `get_variables_by_rdf` rather than all variables.